bothan-coinbase/src/api/types.rs

```diff
@@ -1,6 +1,6 @@
 use serde::{Deserialize, Serialize};
 
-use crate::api::types::channels::ticker::Ticker;
+pub use crate::api::types::channels::ticker::Ticker;
```

Comment:
why pub use here? if you want to export it outside please do so in api.rs

Reply:
fixed